mowgli_etl/pipeline/wdc/parsimonious_parser/wdc_parsimonious_node_visitor.py

```diff
@@ -1,9 +1,28 @@
 from parsimonious.nodes import NodeVisitor
 
+from dataclasses import dataclass, fields
+from typing import Optional
 
 class WdcParsimoniousNodeVisitor(NodeVisitor):
+
+    @dataclass(frozen=True)
+    class __Node:
+        @dataclass
+        class __Entry:
+            value: Optional[float] = None
+            unit: Optional[str] = None
+        width: Optional[__Entry] = __Entry()
```

Comment:
These should be = None and not __Entry(), which will fix serializing them above.